content/matplotlib/concepts/pyplot/terms/bar/bar.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: '.bar()'
+Description: 'Returns a chart/graph that represents categorical data with rectangular bars.'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+Tags:
+  - 'Graphs'
+  - 'Libraries'
+  - 'Matplotlib'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+A **`.bar()`** function returns a chart/graph that represents categorical data with rectangular bars with heights and lengths proportional to the values that they represent. Bars can be plotted vertically or horizontally.
+
+## Syntax
+
+``` pseudo
+plt.bar(x, height, width, bottom, align)
+```
+
+The parameters to the function are:
```

Comment:
The language describing these parameters matches or is almost identical to that in that Matplotlib documentation. Please be aware that all content must be original, we do not accept entries that include content copied from other sites.

Reply:
@caupolicandiaz thanks for your rephrasing the parameters. I wasn't really sure how to rephrase them.